### DISCUSSION TITLE
Sponsorship list

### DISCUSSION THREAD

**Discussion by hidyverse** [resbazaz/festival2024#23] [General]
I'm starting a list here of sponsorship I think we're requesting for documentation records

- DSI
- ICDI
- Jetstream
- UX@UA

**Comment by jcoliver** [comment]
And the Data Cooperative (part of UA Libraries) would likely be able to do something, too.

**Reply by hidyverse** [reply]
Plus already helping with space!

**Comment by julianpistorius** [comment]
@hidyverse Is there anything we still need sponsorship for?

**Reply by hidyverse** [reply]
We have two days of coffee and snacks covered, social event covered, and flyers printed.

If there is anything else we can think of we have had other sponsorship offers!

**Reply by julianpistorius** [reply]
How about sponsoring the virtual conference platform for the first day?

**Reply by hidyverse** [reply]
Totally! Are we feeling brave enough to switch from zoom 3 weeks to showtime?

**Reply by hidyverse** [reply]
Or is zoom not free??

**Reply by julianpistorius** [reply]
Zoom is free if the events are started by UArizona folks (or other people with organization licenses).

And no, I'm not feeling brave enough to switch from Zoom so close to showtime! :sweat_smile:

**Reply by hidyverse** [reply]
Same page! I think I misunderstood this post as under the main sponsorship list. Thank you for bringing up the online venue topic.
